src/overlays/actors/ovl_Obj_Grass_Carry/z_obj_grass_carry.c

```diff
@@ -61,87 +61,89 @@ static ColliderCylinderInit sCylinderInit = {
     { 10, 44, 0, { 0, 0, 0 } },
 };
 
-s16 D_809ABBFC = 0;
-s16 D_809ABC00 = 0;
-s16 D_809ABC04 = 0;
-s16 D_809ABC08 = 0;
+s16 sRotSpeedXTarget = 0;
+s16 sRotSpeedX = 0;
+s16 sRotSpeedYTarget = 0;
+s16 sRotSpeedY = 0;
 
-Vec3f D_809ABC0C[] = {
+static Vec3f sUnitDirections[] = {
     { 0.0f, 0.7071f, 0.7071f },
     { 0.7071f, 0.7071f, 0.0f },
     { 0.0f, 0.7071f, -0.7071f },
     { -0.7071f, 0.7071f, 0.0f },
 };
 
-s16 D_809ABC3C[] = { 0x6C, 0x66, 0x60, 0x54, 0x42, 0x37, 0x2A, 0x26 };
+static s16 sFragmentScales[] = { 108, 102, 96, 84, 66, 55, 42, 38 };
 
 static InitChainEntry sInitChain[] = {
     ICHAIN_F32_DIV1000(gravity, -3200, ICHAIN_CONTINUE),
     ICHAIN_F32_DIV1000(terminalVelocity, -17000, ICHAIN_CONTINUE),
     ICHAIN_VEC3F_DIV1000(scale, 400, ICHAIN_STOP),
 };
 
-void func_809AAE60(ObjGrassCarry* this) {
+void ObjGrassCarry_UpdateVelY(ObjGrassCarry* this) {
     this->actor.velocity.y += this->actor.gravity;
     if (this->actor.velocity.y < this->actor.terminalVelocity) {
         this->actor.velocity.y = this->actor.terminalVelocity;
     }
 }
 
-void func_809AAE94(Vec3f* arg0, f32 arg1) {
-    arg1 += ((Rand_ZeroOne() * 0.2f) - 0.1f) * arg1;
+void ObjGrassCarry_RandScaleVecToZero(Vec3f* vel, f32 scale) {
+    scale += ((Rand_ZeroOne() * 0.2f) - 0.1f) * scale;
 
-    arg0->x -= (arg0->x * arg1);
-    arg0->y -= (arg0->y * arg1);
-    arg0->z -= (arg0->z * arg1);
+    vel->x -= (vel->x * scale);
```

Comment:
seems weirdly inconsistent since the other two are abbr.

Reply:
opted not to make this change since there isn't a hard reason for spelling out velocity

Reply:
`vel`:
![image](https://github.com/zeldaret/mm/assets/7416381/86e868b7-b574-4271-bdc1-3538ce744118)
`velocity`:
![image](https://github.com/zeldaret/mm/assets/7416381/47373d44-6b51-4837-a151-d0ef042be076)

`velocity` is around 8 times more common than `vel`. We should try to avoid increasing the inconsistency in the repository, so please change it to `velocity`

Reply:
I'm not opposed to discussing using the abbreviation `vel` instead of `velocity` since pos/accel are also abbreviated, but I think that's outside the scope of this PR, and consistency is better so it's easier to make that change all at once. So `velocity` seems good just for now